docs/_docs/advanced-topics/building-from-source.md

```diff
@@ -79,8 +78,7 @@ addition, you must have **Node 5.10.0** installed as well.
 
```

Comment:
Same here to point to https://github.com/facebook/nuclide/blob/master/package.json#L114